### DISCUSSION TITLE
GTR3 Pro - calendar is not synced completely

### DISCUSSION THREAD

**Discussion by alevshunov** [zepp-health/zeppos-forum#81] [User Feedback]
I noticed next issue: GTR3 Pro doesn't resync the calendar, any rescheduled and/or cancelled events are untouched and incorrectly displayed on the screen of watches.

Any ideas how to force to update already synced event?

**Comment by htoooth** [comment]
Hi @alevshunov 
I'm sorry for the late reply. Are you saying that the calendar data synchronization is not correct? Would it be possible for you to submit your logs and provide us with your UID so we can look into the issue?

**Comment by ii343hbka** [comment]
Sorry for necroposting but I have the same problem with Balance.
Some events duped (only on watch, at Zepp app it shows correctly). All events from Mondays shifted to Sunday or Tuesday.
What should I do to debug this?

**Reply by MyoungXUE** [reply]
Hello. I will assist you in solving this problem. Please contact me on Discord. I need to collect some information so that I can better solve the problem.

You can follow this link to join Discord. https://docs.zepp.com/docs/guides/community/

**Reply by alevshunov** [reply]
A while ago I used to use a daily basis turn off-on of the calendar sync in the Zepp app to get updated state of the schedule, I have quite a lot of rescheduled events during a day. 
And finally just turned that feature completely off.
